src/components/smart/SystemFields.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <ul data-cy="systemfield-ul">
+    <li data-cy="systemfield-status"><span>Status:</span> {{ props.status || '-' }}</li>
+    <li data-cy="systemfield-aangemaakt-door">
+      <span>Aangemaakt door:</span> {{ props.createdBy || '-' }} op {{ createdAt }}
+    </li>
+    <li data-cy="systemfield-laatst-bewerkt" v-if="props.updatedBy">
+      <span>Laatst bewerkt door:</span> {{ props.updatedBy }} op {{ updatedAt }}
+    </li>
+    <slot data-cy="systemfield-slot"></slot>
```

Comment:
Werkt deze selector? Slot wordt vervangen door hetgeen je in de body van de component zet, dus ik vermoed dat dit element inclusief selector verdwijnt.

Reply:
Jup klopt. Daarom dat die laatste test kijkt naar 4 children in de ul's ipv naar dat systemfield-slot. 
Kzal het weghalen